### PR TITLE
:seedling: Make development with Tilt easier

### LINKED ISSUE #37: Make development with Tilt easier
/kind feature

**Describe the solution you'd like**

Make development with Tilt easier:
- [ ] add button create workload cluster
- [ ] Tilt should create objects like `ClusterStack`, `OpenStackClusterStackReleaseTemplate` and secret with `clouds.yaml` automatically 
- [ ] add `env` variables(SECRET_NAME, CLOUD_NAME, etc.) into .envrc.sample, related to the above sub-task
- [ ] based on the above sub-tasks modify develop readme(#36)

### PR DESCRIPTION
**Pull request by michal-gubricky**
**What this PR does / why we need it**:

Make development with Tilt easier

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #37 

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [x] squash commits
- [ ] include documentation
- [ ] add unit tests

### CODE REVIEW

File: .envrc.sample
Comment (matofeder) [inline .envrc.sample:13]: @michal-gubricky, thanks for this excellent work. I suggest targeting the `main` branch instead of `cspo`. Let's wait for #23, and then we can merge this one directly into the `main`. The reason is that PR#23 has received a thorough review, and merging additional code (unrelated to the review comments) in the middle of the review process might be confusing for the reviewers in #23.

File: config/cspo/cluster.yaml
Comment (chess-knight) [inline config/cspo/cluster.yaml:21]:
```suggestion
    version: v1.27.5
```

File: .envrc.sample
Comment (chess-knight) [inline .envrc.sample:1]: Why do we need to **export K8S_VERSION=1-27** here?

File: docs/develop.md
Comment (chess-knight) [inline docs/develop.md:33]: Where is **Delete Workload Cluster** button?

File: Makefile
Comment (michal-gubricky) [reply Makefile:1]: Yes, I can add it to this PR even though it's not part of issue #37

File: Tiltfile
Comment (michal-gubricky) [reply Tiltfile:207]: Good point, I will rename it.

File: .envrc.sample
Comment (michal-gubricky) [reply .envrc.sample:13]: That sounds reasonable, I will incorporate changes from the review here and then create a new PR, which will target the `main` branch.

File: .envrc.sample
Comment (chess-knight) [reply .envrc.sample:1]: Only k8s_version is unnecessary. KUBECONFIG is used by the tilt

**Review by chess-knight — APPROVED** [review]
LGTM

File: .envrc.sample
Comment (michal-gubricky) [reply .envrc.sample:12]: After #23 was merged into `main`, also the target of this PR was changed automatically to targeting `main` branch

**Pull request merged**
